<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkerInfoDialog</class>
 <widget class="QDialog" name="WorkerInfoDialog">
  <property name="windowTitle">
   <string>작업자 정보</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>220</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <item>
    <widget class="QLabel" name="label_title">
     <property name="objectName">
      <string>label_title</string>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>작업자 정보</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_info">
     <item row="0" column="0">
      <widget class="QLabel" name="label_name_caption">
       <property name="text">
        <string>이름</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_name_value">
       <property name="objectName">
        <string>label_name_value</string>
       </property>
       <property name="text">
        <string>—</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_uid_caption">
       <property name="text">
        <string>카드 UID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_uid_value">
       <property name="objectName">
        <string>label_uid_value</string>
       </property>
       <property name="text">
        <string>—</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_note_caption">
       <property name="text">
        <string>비고</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_note_value">
       <property name="objectName">
        <string>label_note_value</string>
       </property>
       <property name="text">
        <string>—</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_buttons">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_edit">
       <property name="objectName">
        <string>button_edit</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="text">
        <string>수정</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_delete">
       <property name="objectName">
        <string>button_delete</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="text">
        <string>삭제</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_close">
       <property name="objectName">
        <string>button_close</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="text">
        <string>닫기</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
